src/scss/common/_typography.scss

```diff
@@ -1,4 +1,9 @@
+:root{
```

Comment:
Do you really think `_typography` it is a good place for this variable?

`_typography` is for fonts styles only

insert this in modal box scss file

Reply:
Done

Reply:
Are you sure?
![image](https://github.com/IlyaDevN/Horse-club/assets/37267977/b83dd540-2a6a-46ef-b92e-af78209bb5a5)

Reply:
Done